src/components/Navbar.jsx

```diff
@@ -0,0 +1,139 @@
+            src={logoSrc}
+          />
+          <p className="font-bold text-orange-500 uppercase sm:text-xl">
+            Arise
+          </p>
+        </div>
+        <div className="hidden gap-x-10 text-sm lg:flex">
+          {sections.map(({ title, clickHandler }) => (
+            <button key={title} onClick={clickHandler}>
+              <div className="relative ml-auto w-fit group">
+                <p>{title}</p>
+                <span className="hidden absolute w-full h-0.5 bg-gradient-to-r from-yellow-500 to-red-500 group-hover:block" />
+              </div>
+            </button>
+          ))}
+        </div>
+        <div className="flex gap-x-6">
+          {lngs && (
+            <div className="flex relative" ref={translationMenuRef}>
+              <button
+                onClick={() => {
+                  setIsTranslationMenuOpen(!isTranslationMenuOpen);
+                  setIsHamburgerMenuOpen(false);
+                }}
+                ref={translationMenuRef}
+              >
+                <BsTranslate size={24} />
+              </button>
+              {isTranslationMenuOpen && (
+                <div className="absolute -right-6 top-14 p-2 rounded-xl shadow-2xl bg-black/75">
+                  <ul className="w-fit">
+                    {lngs.map(({ FlagIcon, title, clickHandler }) => (
+                      <li
+                        className="flex gap-x-2 items-center py-1 px-2 text-sm rounded-lg hover:bg-gray-300/25"
+                        key={title}
+                        onClick={clickHandler}
+                      >
+                        <FlagIcon className="h-4" />
+                        <p>{title}</p>
+                      </li>
+                    ))}
+                  </ul>
+                </div>
+              )}
+            </div>
+          )}
+          <div className="flex relative lg:hidden" ref={hamburgerMenuRef}>
+            <button
+              onClick={() => {
+                setIsTranslationMenuOpen(false);
+                setIsHamburgerMenuOpen(!isHamburgerMenuOpen);
+              }}
+            >
+              <RxHamburgerMenu size={24} />
+            </button>
+            {isHamburgerMenuOpen && (
+              <div className="absolute -right-6 top-14 p-2 rounded-xl shadow-2xl bg-black/75">
+                <ul className="w-fit">
+                  {sections.map(({ title, clickHandler }) => (
+                    <li
+                      className="flex gap-x-2 items-center py-1 px-2 text-sm rounded-lg min-w-32 hover:bg-gray-300/25"
+                      key={title}
+                      onClick={clickHandler}
+                    >
+                      {title}
+                    </li>
+                  ))}
+                </ul>
+              </div>
+            )}
+          </div>
+          <a
+            className="hidden py-1 px-4 text-sm text-black bg-white rounded-full lg:block hover:bg-gray-100"
```

Comment:
i think you referred to the wrong design. this is the right one for the contact us button:
<img width="748" alt="image" src="https://github.com/user-attachments/assets/68bcf863-37ed-4022-856c-5459707e693b">
when hovered the button looks like this:
<img width="379" alt="image" src="https://github.com/user-attachments/assets/566aff4c-ff45-4bf8-83cb-2a57c6b3535f">

Reply:
this might need reconfirmation from the designers

Reply:
yeah this will be updated in the next revision, it's an initial PR since we were pending design team clarification